src/router/expenses.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+const {
+  getExpenses,
+  createExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../controllers/expenses.js');
+
+const router = express.Router();
+
+router.get('/', getExpenses);
+router.post('/', express.json(), createExpense);

```

Comment:
You should use express.json() middleware globally at app level instead of using it on each route.
please check other places